docs/docs/migration_notes.md

```diff
@@ -9,6 +9,16 @@ Aztec is in full-speed development. Literally every version breaks compatibility
 
 ## 0.87.0
 
+## [Aztec.js/TS libraries]
+
+We've bumped our minimum supported node version to v20, as v18 is now EOL. As a consequence, the deprecated type assertion syntax has been replaced with modern import attributes whenever contract artifact JSONs are loaded:
```

Comment:
```suggestion
We've bumped our minimum supported nodejs version to v20, as v18 is now EOL. As a consequence, the deprecated type assertion syntax has been replaced with modern import attributes whenever contract artifact JSONs are loaded:
```

to avoid confusion with an aztec node